reflex-dom/src/Reflex/Dom/Internal.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Does this end up getting used?

Reply:
In line 33 (`port :: Int <- ...`), yes - but I will replace it with your suggestion which is simpler.

Reply:
Sorry, never mind. I remembered incorrectly. I think it was haskell-stylish that necessitated this extension for some reason. I'll remove it anyway for now.